core/themes/bartik/templates/comment.tpl.php

```diff
@@ -72,9 +75,11 @@
       <span class="new"><?php print $new; ?></span>
     <?php endif; ?>
 
-    <?php print render($title_prefix); ?>
-    <h3><?php print $title; ?></h3>
-    <?php print render($title_suffix); ?>
+    <?php if ($title_options != '2'): ?>
```

Comment:
If it were "title_display", then this would become `if ($title_display)`.

Whether titles are generated or custom doesn't have any meaning here.

Reply:
This comment is dependent upon us coming to an understanding about the previous comment, I think.